modules/tagging/main.tf

```diff
@@ -0,0 +1,8 @@
+resource "ibm_resource_tag" "tag" {
```

Comment:
Make this as root module.
Remove module directory.

Reply:
Removed tagging module directory